src/Libraries/Watch3DNodeModelsWpf/Watch3DNodeModelsWpf.csproj

```diff
@@ -169,10 +169,6 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
-  <ItemGroup>
-    <Content Include="sharpdx_direct3d11_effects_x64.dll" />
```

Comment:
sadly, I may need to revert this for patch compatibility.... @QilongTang what do you think?